README.rst

```diff
@@ -18,45 +18,62 @@ Build and Tests
 Build
 ----------------------------------------------------------------------
 
-Ensure you have cmake 2.8 or later version.
+Prerequisite
+^^^^^^^^^^^^^
+ - Ensure you have cmake 2.8 or later version.
+ - For Windows, require: one of Visual Studio [VS14, VS15, VS16, VS17]
```

Comment:
VS14 needs to be removed. After upgrading arrow to 15.x we don't support VS14 build anymore.

Reply:
Removed all VS 14 in README.rst